.github/workflows/test.yml

```diff
@@ -360,3 +359,48 @@ jobs:
           $env:PATH = "C:\msys64\usr\bin;$env:PATH"
           & "C:\msys64\mingw64\bin\mingw32-make.exe" -C $(Convert-Path test\old\testdir) VERBOSE=1
           $env:PATH = $OldPath
+
+  with-external-deps:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          sudo add-apt-repository ppa:neovim-ppa/stable
+          ./.github/scripts/install_deps.sh
+          sudo apt-get install -y \
+            libluajit-5.1-dev \
+            libmsgpack-dev \
+            libtermkey-dev \
+            libtree-sitter-dev \
+            libunibilium-dev \
+            libuv1-dev \
+            lua-busted \
+            lua-filesystem \
+            lua-inspect \
```

Comment:
off-topic for this PR, but I wonder if we can eliminate lua-inspect (or is it a transitive dependency of some other luarocks package?)

Probably can eliminate others too, since we use Nvim as the test runner now.